pkg/apis/acme/v1/types_issuer.go

```diff
@@ -93,6 +93,14 @@ type ACMEIssuer struct {
 	// Defaults to false.
 	// +optional
 	DisableAccountKeyGeneration bool `json:"disableAccountKeyGeneration,omitempty"`
+
+	// Enables requesting a Not After date on certificates that matches the
+	// duration of the certificate. This is not supported by all ACME servers
+	// like Let's Encrypt. If set to true when the ACME server does not support
+	// it it will create an error on the Order.
+	// Defaults to false.
+	// +optional
+	RequestDuration bool `json:"requestDuration,omitempty"`
```

Comment:
I find the name difficult to understand.
Did you consider something like `PopulateOrderNotAfter` or `EnableNotAfterHandling`? 
It would read better in the code below where this bool is checked.

Also, it might be nice to move this into a `OrderFeatures` sub-struct...or perhaps that's overkill.

Reply:
From previous discussions notAfter should not be used as we do not face that name to the user.
`PopulateOrderNotAfter` I find personally less as what is in the order is quite in depth for ACME. `EnableNotAfterHandling` Thus `EnableDurationHandling` feels a bit weird... i opened a discussion on slack but it had huge success